src/components/ProjectDisciplines.js

```diff
@@ -28,13 +28,15 @@ const mapStateToProps = (state) => ({
   isGuestUser: state.user.isGuestUser,
   isConnected: state.main.isConnected,
   isFetching: state.main.isFetching,
-  pushPrompted: state.user.pushPrompted
+  pushPrompted: state.user.pushPrompted,
```

Comment:
Not sure this is being used. Instead, line 64 uses `this.props.user.pushPrompted`, which seems redundant.